templates/download/risc-v/tab-6.html

```diff
@@ -32,10 +20,15 @@ <h3 class="p-heading--4">Ubuntu Server live installer</h3>
 			  }}
 		</div>
 	</div>
-	<div class="row u-vertically-center u-no-padding">
+	<div class="row">
 		<div class="col-6">
-			<h4>First time installing Ubuntu on QEMU/Unmatched?</h4>
-			<p>Please see the <a href="https://wiki.ubuntu.com/RISC-V/QEMU">How to install Ubuntu on QEMU RISC-V</a> for more information.</p>
+      <hr class="u-no-margin--bottom">
+      <h3 class="p-heading--2 u-sv3">Ubuntu Server live installer</h3>
+			<p>
+				<a class="p-button--positive" href="https://cdimage.ubuntu.com/releases/23.04/release/ubuntu-23.04-live-server-riscv64.img.gz">Download 23.04 live installer</a>
```

Comment:
So this link is duplicated, the other one is right, I believe this one is https://cdimage.ubuntu.com/releases/23.04/release/ubuntu-23.04-live-server-riscv64.img.gz if it's the live installer